pricing-page-exercise/merve/style.css

```diff
@@ -0,0 +1,277 @@
+* {
+    box-sizing: border-box;
+}
+
```

Comment:
```
body {
  margin: 0;
}
``` 

Add this to remove margin from body element and make your header take entire width of the page